lmfit/minimizer.py

```diff
@@ -28,7 +28,7 @@
 from scipy.optimize import minimize as scipy_minimize
 from scipy.stats import cauchy as cauchy_dist
 from scipy.stats import norm as norm_dist
-from scipy.version import version as scipy_version
+import scipy
```

Comment:
I don't see the benefit of importing the complete ```scipy``` module here since we don't use anything from it later on, except for the version. I'd prefer to keep the import as it was and use ```scipy_version``` later on in the comparison. Having said that, I don't if it really matters (e.g., timewise and such) and the other changes look good to me.  @newville what do you think - if you're happy with it, just merge!

Reply:
maybe  `from scipy import __version__ as scipy_version`?  I don't have a strong preference for how it is imported.  I think the important thing is to leave the version components as string.

Reply:
after looking at the [scipy source code](https://github.com/scipy/scipy/blob/master/scipy/__init__.py) I'd prefer the previous solution by @reneeotten which was

```python
from scipy.version import version as scipy_version
```

since in the scipy-package itself  `__version__` is imported as 

```python
from scipy.version import version as __version__
```